src/AutoRest.CSharp/Common/AutoRest/Plugins/NewProjectScaffolding.cs

```diff
@@ -355,6 +355,7 @@ private string GetUnbrandedSrcCSProj()
 
         private static readonly IReadOnlyList<CSProjWriter.CSProjDependencyPackage> _brandedTestDependencyPackages = new CSProjWriter.CSProjDependencyPackage[]
         {
+            new("System.ClientModel", "1.0.0"),
```

Comment:
Need to specify the version for `System.ClientModel` to avoid warning thrown for version downgrade.

Reply:
I thought this wasn't needed any longer?  Shouldn't this pull in transitively from Azure.Core

Reply:
removed